coalib/output/ConsoleInteraction.py

```diff
@@ -952,6 +953,8 @@ def print_bears(bears,
             filename = args.output[0]
             with open(filename, 'w') as fp:
                 fp.write(json_formatted_output)
+        elif not results:
+            print(json_output['bears'])
```

Comment:
I don't think we need to take care of this separately. This way, we'll two different output if we are printing it on console or in file.
Something like this should be fine:
```
{
  bears: []
}
```
And update the tests accordingly.